frontend/src/pages/Assistants/AssistantsDetails.tsx

```diff
@@ -7,45 +7,61 @@ import {
   AsstProperty,
   PropertyContainer,
   PropertyHeading,
+  PropertyList,
   ThumbNail,
+  HouseItem,
 } from './Assistants.styling';
 import { useFetch } from '../../helpers/';
 import img from '../assets/ronald.jpg';
 import { hostname } from 'os';
 
 const AssistantCard = (assistant: any) => {
   return (
-    <AssistantBar>
+    <AssistantBar className={assistant.className}>
       <AsstDetail>
-        <ThumbNail
-          src={img}
-          style={{ width: '96px', height: '96px' }}
-          alt={assistant.full_name}
-        />
-        <div style={{ marginLeft: '1rem' }}>
+        <ThumbNail className='detail-img' src={img} alt={assistant.full_name} />
+        <div className='detail-txt'>
           <h2>{assistant.full_name}</h2>
           <h3>{assistant.address}</h3>
         </div>
       </AsstDetail>
       <AsstProperty>
+        <div className='button-group'>
+          <Button text='Edit Assistant' />
+          <Button text='Go Back' />
+        </div>
         <PropertyContainer>
           <PropertyHeading>
             <h2>Default Properties</h2>
+            <Button className='button-new' text='+ New' />
           </PropertyHeading>
-          {assistant.default_house.map((house: any) => (
-            <p key={house.house_id}>{house.house_name}</p>
-          ))}
-          <Button text='+ Add New Item' />
+          <PropertyList>
+            {assistant.default_house.map((house: any) => (
+              <HouseItem key={house.house_id}>
+                {house.house_name}
+                <span className='hide'>
+                  <i className='fas fa-trash-alt' />

```

Comment:
This would be my only change here, it should probably display a small "x" to be consistent with the rest of the app

Reply:
Lemme do a quick change before we get this merged.